src/filter.js

```diff
@@ -42,7 +42,21 @@ const EXCLUDED_MATERIAL_TYPES = [
 
 export default function (record, earliestCatalogTime) {
 	const leader = record.varFields.find(f => f.fieldTag === '_');
-	const materialType = record.materialType.code.trim();
+
+	const materialType = function () {
```

Comment:
Declare functions at the end of the enclosing function. So this would become `const materialType = getMaterialType()`